interface/ceed-operator.c

```diff
@@ -1469,16 +1469,16 @@ int CeedOperatorSetName(CeedOperator op, const char *name) {
 }
 
 /**
-  @brief View a `CeedOperator`
+  @brief Core logic for viewing a `CeedOperator`
 
-  @param[in] op     `CeedOperator` to view
+  @param[in] op     `CeedOperator` to view brief summary
   @param[in] stream Stream to write; typically `stdout` or a file
 
   @return Error code: 0 - success, otherwise - failure
 
   @ref User
 **/
-int CeedOperatorView(CeedOperator op, FILE *stream) {
+int CeedOperatorView_Core(CeedOperator op, FILE *stream, bool is_full) {
```

Comment:
I feel like reversing the boolean to `is_terse` makes more sense since the associated function is `CeedOperatorTerseView`.

Reply:
I did not do that on purpose so the conditionals below say `if (true) doThing` instead of `if (!true) doThing`

Reply:
Ah, fair enough.